static/src/js/util/collectionMetadata/nativeFormat.js

```diff
@@ -0,0 +1,19 @@
+import { uniq } from 'lodash'
+
+/**
+ * Extracts the native format(s) from the UMM-C metadata
+ * @param {Object} ummJson UMM-C JSON metadata
+ */
+export const buildNativeFormat = (ummJson) => {
+  const { ArchiveAndDistributionInformation = {} } = ummJson
+  const { FileDistributionInformation = [] } = ArchiveAndDistributionInformation
```

Comment:
Does this survive the linter? I thought it'd require the first letter to be lower case?